gengokit/gengokit_test.go

```diff
@@ -172,7 +168,7 @@ func TestApplyTemplateFromPath(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	te, err := newTemplateExecutor(dt, goImportPath)
+	te, err := newTemplateExecutor(dt, goImportPath, goImportPath)
```

Comment:
I actually thought that was a typo up until this comment. I would propose just declaring a second constant.